src/uv_tcp_listen.c

```diff
@@ -275,51 +285,126 @@ static void uvTcpListenCb(struct uv_stream_s *stream, int status)
     assert(rv != 0);
 }
 
+/* Do bind/listen call on the tcp handle */
+static int UvTcpBindListen(struct uv_tcp_s *listener, struct sockaddr *addr)
+{
+    if (uv_tcp_bind(listener, addr, 0) ||
+        uv_listen((uv_stream_t *)listener, 1, uvTcpListenCb)) {
+        return RAFT_IOERR;
+    }
+    return 0;
+}
+
+/* Create a tcp handle and do bind/listen for each IP */
+static int UvTcpListenOnMultipleIP(struct raft_uv_transport *transport,
```

Comment:
Same: should be `uvTcpListenOnMultipleIP`

Reply:
Was not aware of this convention. Fixed it for all newly added helper function in uv_tcp_listen.c